docs/book/features/vsphere_csi_migration.md

```diff
@@ -140,23 +141,33 @@ To try out vSphere CSI migration in beta for vSphere plugin, perform the followi
             clusterrolebinding.rbac.authorization.k8s.io/vsphere-csi-webhook-role-binding created
             deployment.apps/vsphere-csi-webhook created
 
-6. Enable feature flags `CSIMigration` and `CSIMigrationvSphere`
+7. Enable feature flags `CSIMigration` and `CSIMigrationvSphere`
 
    - `CSIMigrationvSphere` flag enables shims and translation logic to route volume operations from the vSphere in-tree plugin to vSphere CSI plugin. Supports falling back to in-tree vSphere plugin if a node does not have a vSphere CSI plugin installed and configured.
    - `CSIMigrationvSphere` requires `CSIMigration` feature flag to be enabled. This flag is enabling CSI migration on the Kubernetes Cluster.
 
-    Steps:
-   - Enable feature flags `CSIMigration` and `CSIMigrationvSphere` on `kube-controller` and `kubelet` on all primary nodes.
+    6.1 Steps for the control plane node(s)
```

Comment:
7.1?

Reply:
on the preview for this change, this is not aligned properly - https://deploy-preview-795--kubernetes-sigs-vsphere-csi-driver.netlify.app/features/vsphere_csi_migration.html

Reply:
Fixed (I think)